src/Ssl/Generator/KeyOption.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+/*
+ * This file is part of the Acme PHP project.
+ *
+ * (c) Titouan Galopin <galopintitouan@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace AcmePhp\Ssl\Generator;
+
+abstract class KeyOption
```

Comment:
Shouldn't this be an interface instead? Not sure why an empty abstract class can be useful.